napari/plugins/hub.py

```diff
@@ -55,9 +55,10 @@ def hub_plugin_info(
         try:
             with request.urlopen(anaconda_api) as resp_api:
                 anaconda_info = json.loads(resp_api.read().decode())
-                if version in anaconda_info.get("versions", []):
-                    is_available_in_conda_forge = True
-
+                versions = anaconda_info.get("versions", [])
+                is_available_in_conda_forge = True
+                if version not in versions:
+                    version = versions[-1]
```

Comment:
```suggestion
                versions = anaconda_info.get("versions", [])
                is_available_in_conda_forge = bool(versions)
                if is_available_in_conda_forge and version not in versions:
                    version = versions[-1]
```

Are you sure that anaconda API return versions in sorted order? Or sort steep is missed?

Reply:
>Are you sure that anaconda API return versions in sorted order? Or sort steep is missed?

Yes they are returned in sorted order

Reply:
Nice, but still, the code needs to handle an empty list properly.

Reply:
Fixed and added tests.